_applications/var.md

```diff
@@ -41,44 +41,46 @@ install and run the notebook image.
 ## Prerequisites
 
 The easiest way to get a Spark cluster going in your OpenShift project is to
-use [`oshinko-deploy.sh`][odsh]. This provides a streamlined installation for
+use [the OpenShift template](https://radanalytics.io/get-started#quickstart-with-oshinko) 
+provided on the "Getting Started page".  This provides a streamlined installation for
 the Oshinko cluster manager into your project. Oshinko is a service to manage
 Spark clusters for OpenShift projects.
 
-[odsh]: https://github.com/radanalyticsio/oshinko-webui/blob/master/tools/oshinko-deploy.sh
-
 We'll need to use a special Spark worker image for this application. This
 worker image has a small amount of historical stock return data stored in its
 filesystem. For a real application, you'd want to get data from persistent
 storage, from a database, or from an object store service. For a demo, though,
 it's much easier to package up a small amount of data where each worker can get
 it.
 
-In order to get this special Spark worker image, we'll use the `-s` flag to
-`oshinko-deploy.sh`:
-
-`oshinko-deploy.sh -s willb/var-spark-worker`
+In order to get this special Spark worker image, we'll need to pass a special option 
+to the Oshinko template when we install it in our project.  Set the `OSHINKO_CLUSTER_IMAGE` 
+variable to `radanalyticsio/workshop-notebook`.  This image is based on the default cluster
```

Comment:
i must have done this wrong, because my master & workers went into a crashloopbackoff and eventually the deployment failed.

i did a 'oc new-app oshinko-webui', went to the webui and when 'deploy'ing a cluster hit advanced options and specified the workshop-notebook instead of openshift-spark.

the resulting master and worker pods kept trying to start a notebook, which in retrospect is obvious because the command for the image starts jupyter instead of a spark master or worker.

what did i do wrong?